src/content/puck.ts

```diff
@@ -41,8 +43,17 @@ type ClickState =
       kind: 'firstpointerdown';
       // This is the timeout we use to detect if it's a drag or not
       timeout: number;
```

Comment:
Should we rename this to `tapTimeout`?

Reply:
It complicates the typing of `clearClickTimeout` above slightly to have both this and `doubleClickTimeout`, but I think it's worth the increased clarity. Think I'll go with `clickTimeout` for consistency though.